### PR TITLE
Erfi submit dupe prevention

### PR DESCRIPTION
**Pull request by mlsamuelson**
### Description

Formik's `isSubmitting` wasn't successfully disabling the RFI submit button when clicked. Lag time maybe? `submitCount` works better in testing.
git

### CODE REVIEW

**Comment by mlsamuelson** [comment]
https://formik.org/docs/guides/form-submission

Another idea about why `isSubmitting` wasn't working out here could be that the value is set back to false faster than the component updates.  And the variability of all this could account for variations in what I was seeing when investigating the issue. To avoid timing issues, `submitCount` ought to be more effective.

File: packages/app-rfi/src/components/stepper/RfiStepperButtons.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const RfiStepperButtons = ({
               <Button
                 type="submit"
                 className="rfi-submit btn btn-primary"
-                disabled={!!isSubmitting}
+                disabled={submitCount > 0}
```
Comment (scott-williams-az) [inline packages/app-rfi/src/components/stepper/RfiStepperButtons.js]: SubmitCount is increased each time the form tries to submit or go to the next page. One page variant click submit and the form will be stuck 
image

Comment (mlsamuelson) [reply packages/app-rfi/src/components/stepper/RfiStepperButtons.js]: PR now uses the `totalSteps` in the disabled logic to account for steps getting logged as submits.

Comment (mlsamuelson) [reply packages/app-rfi/src/components/stepper/RfiStepperButtons.js]: Needs more work: Paging back and forth through the form can exceed the submit count.

**Comment by mlsamuelson** [comment]
Incorporated @scott-williams-az improvements that use useState() to make the submit disabling more reliable.

**Review by davidornelas11 — APPROVED** [review]
Tested in a webspark environment and it works as expected. The button is disabled and cannot be pressed again, even on keyboard.

**Pull request merged**